examples/all_the_things.py

```diff
@@ -99,16 +102,16 @@ def teardown(test):
       # but you can include any metadata fields.
       test_name='MyTest', test_description='OpenHTF Example Test',
       test_version='1.0.0')
-  test.add_output_callbacks(htf.output.callbacks.OutputToFile(
+  test.add_output_callbacks(callbacks.OutputToFile(
       './{dut_id}.{metadata[test_name]}.{start_time_millis}.pickle'))
   test.add_output_callbacks(
-      htf.output.callbacks.json_factory.OutputToJSON(
+      json_factory.OutputToJSON(
```

Comment:
I would move this up to the previous line since it all probably fits on one now.